packages/client/src/runtime/core/init/validateEngineInstanceConfig.ts

```diff
@@ -76,32 +63,14 @@ export function validateEngineInstanceConfig({
   const isUsingDriverAdapters = Boolean(adapter)
   const isAccelerateUrlScheme = isUsingPrismaAccelerate || isUsingPrismaPostgres
 
-  if (
-    !isUsingDriverAdapters &&
-    copyEngine &&
-    isAccelerateUrlScheme &&
-    targetBuildType !== 'client' &&
-    targetBuildType !== 'wasm-compiler-edge'
-  ) {
-    pushWarning([
-      'recommend--no-engine',
-      'In production, we recommend using `prisma generate --no-engine` (See: `prisma generate --help`)',
-    ])
-  }
-
-  const isAccelerateConfigured = isAccelerateUrlScheme || !copyEngine
+  const isAccelerateConfigured = isAccelerateUrlScheme
 
   if (isUsingDriverAdapters && isAccelerateConfigured) {
     if (isAccelerateUrlScheme) {
```

Comment:
redundant `if`, it is now covered by the condition above (`isAccelerateConfigured` and `isAccelerateUrlScheme` are the same variable)